chirp/drivers/baofeng_uv17Pro.py

```diff
@@ -865,7 +868,7 @@ def get_features(self):
             "DTCS->DTCS"]
         rf.valid_skips = self.SKIP_VALUES
         rf.valid_dtcs_codes = self.DTCS_CODES
-        rf.memory_bounds = (0, self.CHANNELS - 1)
+        rf.memory_bounds = (1, self.CHANNELS)
```

Comment:
This is a change for all the affected radios right? Was this an error in the initial implementation? It's obviously really unideal to ever change this on a whim...

Reply:
It affects all radios. People were complaining that the channel numbers in CHIRP started at 0, where in the radio it starts at 1. So now it shows channel 1 as 1 in CHIRP. Only affects the way things are displayed.

Reply:
Yeah, I know it's just the display, and it should be right, it's just unfortunate to change it after the driver has been merged.

Reply:
I agree... I didn't realise it was possible. Does it introduce a problem?

Reply:
I agree.. Still learning the CHIRP ways. :-)